mcp/src/main/java/io/modelcontextprotocol/client/McpAsyncClient.java

```diff
@@ -780,16 +780,15 @@ private NotificationHandler asyncLoggingNotificationHandler(
 	 * @return A Mono that completes when the logging level is set.
 	 * @see McpSchema.LoggingLevel
 	 */
-	public Mono<Void> setLoggingLevel(LoggingLevel loggingLevel) {
+	public Mono<Object> setLoggingLevel(LoggingLevel loggingLevel) {
```

Comment:
This is a backwards incompatible change. I understand we can classify this as a bug?

Reply:
yes it is a bug

Reply:
Actually, I think we should not change this signature. Internally there is an empty result, agreed. But I think we can leave the API as the empty response is not useful to the user.